interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -282,6 +282,17 @@ namespace cling {
     ///\param[in] llvmdir - ???
     ///\param[in] noRuntime - flag to control the presence of runtime universe
     Interpreter(int argc, const char* const *argv, const char* llvmdir = 0, bool noRuntime = false);
+
+    ///\brief Constructor for Interpreter.
+    ///
+    ///\param[in] argc - no. of args.
+    ///\param[in] argv - arguments passed when driver is invoked.
+    ///\param[in] llvmdir - ???
+    ///\param[in] noRuntime - flag to control the presence of runtime universe
+    ///\param[in] originalInterpreter - the master interpreter of this interpreter
+    Interpreter(Interpreter *originalInterpreter,int argc, const char* const *argv,
```

Comment:
Can we pass in an `Interpreter&` instead of a pointer? This is required to be non null, passing a reference signals that.